app/models/couch.rb

```diff
@@ -4,7 +4,7 @@ class Couch < ApplicationRecord
   has_many :nights
   has_many :reservations, through: :nights
 
-  scope :in_city, -> (city) { where(city: city) }
+  scope :in_city, -> (city) { where("lower(city) = ?", city.downcase) }
```

Comment:
@njgheorghita I totally agree. I went with the lower clause for now because it's the only search term we have. I think citext becomes advantageous if we want to search for multiple attributes in a single WHERE clause.